README.md

```diff
@@ -244,71 +271,75 @@ AwsSsmEnv.load!(
 
 #### other
 
-`AwsSsmEnv::NamingStrategy`を実装したクラスのインスタンス、もしくは`parse_name`メソッドを持つ  
-インスタンスを指定した場合はそのインスタンスをそのまま利用する。
+If you specify an instance of a class that implements `AwsSsmEnv::NamingStrategy` in` fetch`,
+or an instance with a `parse_name` method, use that instance as is.
+
 
 ### path: [String]
 
-`fetch`に何も指定していない場合、もしくは`:path`を指定した場合は必須となる。  
-パラメータを取得するパス階層を指定する。  
-下の例では`/myapp/web/production`直下のパラメータが取得される。
+It is required if nothing is specified for `fetch` or if `:path` is specified.  
+Specify the path hierarchy for acquiring parameters.  
+In the example below, the parameter immediately under `/myapp/web/production` is acquired.
 
 ```ruby
 AwsSsmEnv.load(path: '/myapp/web/production')
 ```
 
 #### recursive: [Boolean]
 
-`fetch`に何も指定していない場合、もしくは`:path`を指定した場合に利用する。  
-指定したパス階層以下のパラメータをすべて取得する。  
-下の例では`/myapp/web/production`以下すべてのパラメータが取得される。
+It is required if nothing is specified for `fetch` or if `:path` is specified.  
+Acquires all parameters below the specified path hierarchy.  
```

Comment:
Two points which I pointed out for the part of `:path` are also applicable to this paragraph.

For example:

> Required if nothing is specified for `fetch` or if `:path` is specified.
By this option, it acquires all parameters under the specified path hierarchy.

Reply:
I'm mistake, this parameter is not required.  
`recursive` parameter is `false`, when it not specified.

Reply:
@progrhyme 

I propose the following, is it correct?

> Used when nothing is specified for `fetch` or if `:path` is specified.
> Acquires all parameters below the specified path hierarchy.
> If nothing is specified this parameter, it is treated as `false`(one level).

Reply:
Make sense.
But first sentence would be better if it goes like ...

> Used when no parameter is specified for `:fetch` option or when `:path` is specified.